### PR TITLE
Release v0.9.3

### PR DESCRIPTION
**Pull request by cgilmour**
This updates the installer to deploy Romana v0.9.3.

### CODE REVIEW

**Comment by cgilmour** [comment]
Kubernetes test results:
```
2016-09-05 09:18:56 (ip-192-168-99-10:clustertests) Test #1: Check that Kubernetes services are running on master : PASSED
2016-09-05 09:18:57 (ip-192-168-99-10:clustertests) Test #2: Check that Romana services are running on master : PASSED
2016-09-05 09:18:58 (ip-192-168-99-10:clustertests) Test #3: Check that Kubernetes services are running on minion : PASSED
2016-09-05 09:18:59 (ip-192-168-99-10:clustertests) Test #4: Check that Romana services are running on minion : PASSED
2016-09-05 09:19:00 (ip-192-168-99-10:clustertests) Test #6: Pull docker containers used in later tests : PASSED
2016-09-05 09:19:18 (ip-192-168-99-10:clustertests) Test #8: Check namespace creation triggers romana tenant creation : PASSED
2016-09-05 09:19:19 (ip-192-168-99-10:clustertests) Test #7: Check that kubernetes can launch some pods : PASSED
2016-09-05 09:19:34 (ip-192-168-99-10:clustertests) Test #9: End-to-End test checking default-allow, isolation and policy-allow : PASSED
2016-09-05 09:18:58 (ip-192-168-99-11:clustertests) Test #3: Check that Kubernetes services are running on minion : PASSED
2016-09-05 09:18:59 (ip-192-168-99-11:clustertests) Test #4: Check that Romana services are running on minion : PASSED
2016-09-05 09:19:00 (ip-192-168-99-11:clustertests) Test #6: Pull docker containers used in later tests : PASSED
```
Demo script: passed.

**Comment by cgilmour** [comment]
Devstack test results:
```
2016-09-05 09:35:33 (ip-192-168-99-10:clustertests) Test #1: Check that openstack services are present : PASSED
2016-09-05 09:35:38 (ip-192-168-99-10:clustertests) Test #2: Check that Romana services are running on controller node : PASSED
2016-09-05 09:35:39 (ip-192-168-99-10:clustertests) Test #3: Check that Romana services are running on compute nodes : PASSED
2016-09-05 09:35:41 (ip-192-168-99-10:clustertests) Test #4: Check that VMs can be created on each compute node : PASSED
2016-09-05 09:36:00 (ip-192-168-99-10:clustertests) Test #5: End-to-End test checking ping and SSH for VMs in same segment. : PASSED
2016-09-05 09:35:39 (ip-192-168-99-11:clustertests) Test #3: Check that Romana services are running on compute nodes : PASSED
```

File: romana-install/romana-setup [hunk 1]
```diff
@@ -147,6 +147,7 @@ if (( $# > 0 )); then
 				shift
 				;;
 			--core-branch)
+			        core_source="github"
```
Comment (jbrendel) [inline romana-install/romana-setup]: What was this change all about?

**Comment by jbrendel** [comment]
Looks ok. Does it work? :-)

File: romana-install/romana-setup (hunk 1 above)
Comment (cgilmour) [inline romana-install/romana-setup]: Greg wanted that option to implicitly mean `--source` too. (Builds from source instead of downloaded binaries.)

**Comment by cgilmour** [comment]
Retest.

Kubernetes test result:
2016-09-08 05:47:34 (ip-192-168-99-10:clustertests) Test #1: Check that Kubernetes services are running on master : PASSED
2016-09-08 05:47:35 (ip-192-168-99-10:clustertests) Test #2: Check that Romana services are running on master : PASSED
2016-09-08 05:47:36 (ip-192-168-99-10:clustertests) Test #3: Check that Kubernetes services are running on minion : PASSED
2016-09-08 05:47:37 (ip-192-168-99-10:clustertests) Test #4: Check that Romana services are running on minion : PASSED
2016-09-08 05:47:38 (ip-192-168-99-10:clustertests) Test #6: Pull docker containers used in later tests : PASSED
2016-09-08 05:47:58 (ip-192-168-99-10:clustertests) Test #8: Check namespace creation triggers romana tenant creation : PASSED
2016-09-08 05:47:59 (ip-192-168-99-10:clustertests) Test #7: Check that kubernetes can launch some pods : PASSED
2016-09-08 05:48:12 (ip-192-168-99-10:clustertests) Test #9: End-to-End test checking default-allow, isolation and policy-allow : PASSED
2016-09-08 05:47:37 (ip-192-168-99-11:clustertests) Test #3: Check that Kubernetes services are running on minion : PASSED
2016-09-08 05:47:37 (ip-192-168-99-11:clustertests) Test #4: Check that Romana services are running on minion : PASSED
2016-09-08 05:47:38 (ip-192-168-99-11:clustertests) Test #6: Pull docker containers used in later tests : PASSED

Devstack test result:
2016-09-08 05:51:59 (ip-192-168-99-10:clustertests) Test #1: Check that openstack services are present : PASSED
2016-09-08 05:52:03 (ip-192-168-99-10:clustertests) Test #2: Check that Romana services are running on controller node : PASSED
2016-09-08 05:52:05 (ip-192-168-99-10:clustertests) Test #3: Check that Romana services are running on compute nodes : PASSED
2016-09-08 05:52:06 (ip-192-168-99-10:clustertests) Test #4: Check that VMs can be created on each compute node : PASSED
2016-09-08 05:52:26 (ip-192-168-99-10:clustertests) Test #5: End-to-End test checking ping and SSH for VMs in same segment. : PASSED
2016-09-08 05:52:05 (ip-192-168-99-11:clustertests) Test #3: Check that Romana services are running on compute nodes : PASSED

**Pull request merged**